src/modules/fancyzones/lib/ZoneSet.cpp

```diff
@@ -255,6 +255,29 @@ ZoneSet::ZonesFromPoint(POINT pt) const noexcept
             return max(rect.bottom - rect.top, 0) * max(rect.right - rect.left, 0);
         };
 
+        auto getCenter = [](auto zone) {
+            RECT rect = zone->GetZoneRect();
+            return POINT{ (rect.right + rect.left) / 2, (rect.top + rect.bottom) / 2 };
+        };
+        auto pointDifference = [](POINT pt1, POINT pt2) {
+            return (pt1.x - pt2.x) * (pt1.x - pt2.x) + (pt1.y - pt2.y) * (pt1.y - pt2.y);
+        };
+        auto distanceFromCenter = [&](auto zone) {
+            POINT center = getCenter(zone);
+            return pointDifference(center, pt);
+        };
+        auto sensitivity = 75;
```

Comment:
Please, don't use magic numbers. Specify a constant in the namespace at the top of the `ZoneSet.cpp`